pandas/core/api.py

```diff
@@ -28,8 +28,18 @@
 from pandas.tseries.tdi import TimedeltaIndex, Timedelta
 from pandas.tseries.period import Period, PeriodIndex
 
-# legacy
-import pandas.core.datetools as datetools
+# see gh-14094.
+from pandas.util.depr_module import _DeprecatedModule
+
+_alts = ['pandas.tseries.tools', 'pandas.tseries.offsets',
+         'pandas.tseries.frequencies']
+_removals = ['day', 'bday', 'businessDay', 'cday', 'customBusinessDay',
+             'customBusinessMonthEnd', 'customBusinessMonthBegin',
```

Comment:
hmm can't u just introspect the module when u r creating _DeprecatedModule? iow look at all of the symbols the dir() pulls up and use them? rather than listing them all here (just seems simpler / less error prone)